test/arrayops.jl

```diff
@@ -689,6 +689,28 @@ end
     @test isequal(cumsum(A,dims=2),A2)
     @test isequal(cumsum(A,dims=3),A3)
 
+    R = repeat([1, 2, 3], 1:3)
+    @test R == [1, 2, 2, 3, 3, 3]
+    R = repeat([1 2 3], 2, 1:3)
+    @test R == [1 2 2 3 3 3;
+                1 2 2 3 3 3]
+    R = repeat([1 2;
+                3 4], 2, 1:2)
+    @test R == [1 2 2;
+                3 4 4;
+                1 2 2;
+                3 4 4]
+    R = repeat([1 2;
+                3 4], 1:2, 2)
+    @test R == [1 2 1 2;
+                3 4 3 4;
+                3 4 3 4]
+    R = repeat([1 2;
+                3 4], 1:2, 1:2)
+    @test R == [1 2 2;
+                3 4 4;
+                3 4 4]
+
```

Comment:
Can you check what happens when zeros are passed?

Reply:
I've added some tests for that. E.g.
```julia
julia> R = repeat([1 2;
                   3 4], [0, 2], [0, 2])
2×2 Array{Int64,2}:
 4  4
 4  4
```